.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Node
+
+on: [push]
```

Comment:
I guess we need `pull_request` here as well?

Reply:
Yes, good point.